components/net_traits/lib.rs

```diff
@@ -1,17 +1,21 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
```

Comment:
Isn't there supposed to be a newline between the license header and the first `#!` line?